torch/lib/c10d/GlooDeviceFactory.cpp

```diff
@@ -8,6 +8,10 @@
 #include <gloo/transport/tcp/device.h>
 #endif
 
+#if GLOO_HAVE_TRANSPORT_TCP_TLS
```

Comment:
where is this defined?

Reply:
Defined in [gloo/config.h.in:37](https://github.com/facebookincubator/gloo/pull/260/files#diff-36d25b293ab5437090249cefcea89a2bR37) and set in [gloo/CMakeLists.txt:81-85](https://github.com/facebookincubator/gloo/pull/260/files#diff-3ec7c720473f942e710c60b0ea8d794fR82)